fb_instant_articles.module

```diff
@@ -83,3 +83,28 @@ function fb_instant_articles_html_head_alter(&$head_elements) {
     }
   }
 }
+
+/**
+ * Implements hook_init().
+ *
+ * @todo Update to use a custom DrupalFBInstantArticlesLogger once
+ *   @link https://github.com/facebook/facebook-instant-articles-sdk-php/issues/14 this SDK issue @endlink is in resolved.
+ */
+function fb_instant_articles_init() {
+  $enable_transformer_logging = variable_get('fb_instant_articles_enable_transformer_logging');
```

Comment:
@m4olivei I missed this one, thanks. Yes, the idea is to enable **all** logging. When/if we find more appenders in the Instant Articles SDK we can register them all here. That can be a follow-up as needed though. For now, fixed the variable in ec6e6cd.